modules/ml/src/svm.cpp

```diff
@@ -1459,7 +1459,7 @@ class SVMImpl CV_FINAL : public SVM
                     {
                         int cj = class_ranges[j+1] - class_ranges[j];
                         if( nu*(ci + cj)*0.5 > std::min( ci, cj ) )
-                            // TODO: add some diagnostic
```

Comment:
CV_Error raises exception. The method would interrupt execution here instead of false result.
Alternative solution is just log message like
```
 CV_LOG_ERROR(NULL, "Training cases incompatible with nu parameter—try a lower value.");
```